### PR TITLE
add test coverage around sync plan enable/disable

### PR DESCRIPTION
**Pull request by jameerpathan111**
Fixes issue #51 
Test Result:

```
$ pytest --ansible-host-pattern satellite --ansible-user=root  --ansible-inventory testfm/inventory tests/test_advanced.py -k test_positive_sync_plan_disable_enable -svvv
========================================================================= test session starts ==========================================================================
platform linux -- Python 3.6.4, pytest-3.6.1, py-1.6.0, pluggy-0.6.0 -- /home/jpathan/projects/testfm/env2/bin/python
cachedir: .pytest_cache
ansible: 2.6.3
plugins: services-1.2.1, mock-1.6.3, ansible-2.0.1
collected 15 items / 14 deselected                                                                                                                                     

tests/test_advanced.py::test_positive_sync_plan_disable_enable 2019-04-02 17:45:40,411 - testfm.log - INFO - Running ForemanMaintain::Scenario
================================================================================
disable active sync plans: 
\ Total 1 sync plans are now disabled.                                [OK]      
--------------------------------------------------------------------------------
2019-04-02 17:45:58,175 - testfm.log - INFO - Running ForemanMaintain::Scenario
================================================================================
re-enable sync plans: 
- Total 1 sync plans are now enabled.                                 [OK]      
--------------------------------------------------------------------------------
PASSED

=============================================================== 1 passed, 14 deselected in 63.28 seconds ===============================================================
```

### CODE REVIEW

File: tests/test_advanced.py [hunk 1]
```diff
@@ -341,38 +364,26 @@ def test_positive_sync_plan_enable(ansible_module):
...
-    :steps:
-        1. Run foreman-maintain advanced procedure run
-        sync-plans-disable
-
-    :expectedresults: procedure sync-plans-disable should work.
-
-    :CaseImportance: Critical
-    """
-    contacted = ansible_module.command(Advanced.run_sync_plans_disable())
+        assert "Total 1 sync plans are now disabled." in result["stdout"]
```
Comment (ntkathole) [inline tests/test_advanced.py]: @jameerpathan111 this wont be good idea to make the automation system dependant. Do not assert the count. Good way will be asserting id of sync-plan in data.yml.

Comment (jameerpathan111) [reply tests/test_advanced.py]: > @jameerpathan111 this wont be good idea to make the automation system dependant. Do not assert the count. Good way will be asserting id of sync-plan in data.yml.

makes sense

**Comment by jameerpathan111** [comment]
Test Result:

```
$ pytest --ansible-host-pattern satellite --ansible-user=root  --ansible-inventory testfm/inventory tests/test_advanced.py -k test_positive_sync_plan_disable_enable -svvv
========================================================================= test session starts ==========================================================================
platform linux -- Python 3.6.4, pytest-3.6.1, py-1.6.0, pluggy-0.6.0 -- /home/jpathan/projects/testfm
ansible: 2.6.3
plugins: services-1.2.1, mock-1.6.3, ansible-2.0.1
collected 15 items / 14 deselected                                                                                                                                     

tests/test_advanced.py::test_positive_sync_plan_disable_enable 2019-04-05 12:18:57,254 - testfm.log - INFO - Running ForemanMaintain::Scenario
================================================================================
disable active sync plans: 
\ Total 1 sync plans are now disabled.                                [OK]      
--------------------------------------------------------------------------------
2019-04-05 12:19:23,327 - testfm.log - INFO - Running ForemanMaintain::Scenario
================================================================================
re-enable sync plans: 
\ Total 1 sync plans are now enabled.                                 [OK]      
--------------------------------------------------------------------------------
PASSED

=============================================================== 1 passed, 14 deselected in 98.50 seconds ===============================================================
```

File: tests/test_advanced.py [hunk 2]
```diff
@@ -12,11 +15,36 @@
...
+    sync_plan_name = gen_string('alpha')
+    sync_date = datetime.datetime.today().strftime('%Y-%m-%d')
+    setup = ansible_module.command(
+        'hammer sync-plan create --name {0} --enabled true \
+        --interval "weekly" --sync-date {1} --organization-id 1'.format(
+            sync_plan_name, sync_date))
+    for result in setup.values():
+        assert result["rc"] == 0
+    setup = ansible_module.command(
+        'hammer sync-plan info --name {0} --organization-id 1'.format(
```
Comment (ntkathole) [inline tests/test_advanced.py]: Have you checked with json or csv format of hammer to avoid splitting [int(s) for s in sync_id.split() ?

**Pull request merged**